btree/btree.c

```diff
@@ -4540,11 +4558,15 @@ int64_t btree_iter_kv_tick(struct m0_sm_op *smop)
 			 * other thread which has lock is working on the same
 			 * node(lev->l_node) which is pointed by current thread.
 			 */
-			if (!node_isvalid(s.s_node)) {
+			lev->l_sibling->n_skip_rec_count_check = true;
+			if (!node_isvalid(s.s_node) || (oi->i_pivot > 0 &&
```

Comment:
you can use `lev->l_sibling` in place of `s.s_node` for consistency.

Reply:
okay